includes/admin/views/experiments/experiments.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Admin Experiments View
+ *
+ * @package    Secure Custom Fields
+ * @since      6.4.3
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+// Get current screen and experiments.
+$screen      = get_current_screen();
+$experiments = acf()->admin_experiments->get_experiments();
+?>
+<div class="wrap" id="scf-admin-experiments">
+	<h1 class="wp-heading-inline"><?php esc_html_e( 'Experiments', 'secure-custom-fields' ); ?></h1>
```

Comment:
I really think we should land on one naming schema and call it `Experiments` or `Beta features` consistently. This is what I see at the moment:

<img width="797" alt="Screenshot 2025-04-25 at 11 38 50" src="https://github.com/user-attachments/assets/2d99ade3-0aaa-4417-a6c7-d9c38bd230fc" />

Reply:
Agreed, let's go with Beta Features consistently.

Reply:
Should we do it with variables and filenames too? Now is the best momento to do it.